modules/adagioBidAdapter.js

```diff
@@ -108,6 +110,9 @@ export function getAdagioScript() {
       // It's an antipattern regarding the TCF2 enforcement logic
       // but it's the only way to respect the user choice update.
       window.localStorage.removeItem(ADAGIO_LOCALSTORAGE_KEY);
+      // Extra data from external script.
+      // This key is removed only if localStorage is not accessible.
+      window.localStorage.removeItem('adagio');
```

Comment:
I've done some tests to test this line. 
It's impossible to disable the localstorage on chrome - either cookies & localstorage, but not localstorage only. 
I succeded to disable localstorage on firefox but then this code will fail because, of course, localstorage is inaccessible. So, this will result in an exception. Am I wrong?

Reply:
Hello, 
Let me know if I misunderstood but You do not have to try to test the HTML localstorage feature availability itself. 

This is about looking for the capability to read/write the localstorage through Prebid StorageManager which depends on the user choice in the CMP.

(This is the exact same behavior used on the line above the one you want to test)

Let me know if you need more information.